examples/app-example-androidjunitrunner/app/src/androidTest/java/com/example/screenshots/ScreenshotViewAction.java

```diff
@@ -0,0 +1,31 @@
+package com.example.screenshots;
+
+import android.support.test.espresso.UiController;
+import android.support.test.espresso.ViewAction;
+import android.view.View;
+import com.facebook.testing.screenshot.Screenshot;
+import org.hamcrest.Matcher;
+
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+
+public final class ScreenshotViewAction implements ViewAction {
+    public static ViewAction screenshot() {
```

Comment:
We should pass in a name here so we're not left with "unknown" as the image name